javascript/javascript-import-and-require.md

```diff
@@ -0,0 +1,149 @@
+# JavaScript - Require & Import
+
+## Project Time
+
+About 40 Minutes
+
+- video and reference material: 20 minutes
+- Understanding topics: 10 minutes 
+- Pratice challenge: 10 minutes
+
+## Prerequisites
+
+- JavaScript 7
+
+## Motivation
+
+Javascript is one of the most used language. It is lightweight, Interpreted and Just-in-time compiled programming language with the first-class function. While it is most well-known as the scripting language for a webpage, Many no-browser environments also use it. JavaScript is a prototype-based, multi-paradigm, single-threaded, dynamic language, supporting object-oriented, imperative and declarative style.
+
+Here in Javascript, Import and Require is one of the important part and  also crucial part of it. It major useful function, It help us to divide the files into different modules and divide our code modules in systematic ways.
+
+```
+Porject:
+        assert
+          |
+          --> index.js
+        img
+          |
+          --> file.png
+        css
+          |
+          --> index.css
+          
+        index.html
+```
+Now you will get one question in your mind that How we use index.js in index.html, Now Import and Require will come is use here and using these function, you can organize your work in systematic way.

```

Comment:
JS doesn't come into an HTML file with require or import... it uses a <script> tag. 

Import and require are used to bring code **into** a JS file.